src/components/NavBar.js

```diff
@@ -23,7 +23,7 @@ const NavBar = () => {
         {links.map((link) => (
           <li key={link.id}>
             <Link to={link.path}>
-              {link.text}
+              {link.text }
```

Comment:
- [ ] Please let's stick as closely as possible to the design guideline by adding the user icon on the far right of the navbar as shown in the attached image below 👇 
### Zeplin Design
![image](https://user-images.githubusercontent.com/46839548/201467950-addbafd9-b8c4-4714-a833-eb73d94e3a3b.png)
### Your Design
![image](https://user-images.githubusercontent.com/46839548/201467998-2c2a611c-04ed-4c48-96ea-f358de731696.png)